### ISSUE TITLE
Update .net references to version 6.

### ISSUE THREAD

**Issue by OranguTech** [dotnet/docs#29644]
[Enter feedback here]
You probably want to update references to .NET version from 5 to 6.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3ad87897-30cc-e639-b91a-b3800269f7fe
* Version Independent ID: 9d3015b4-cba1-0956-9ec7-4154dbb74d93
* Content: [.NET (and .NET Core) - introduction and overview](https://docs.microsoft.com/en-us/dotnet/core/introduction)
* Content Source: [docs/core/introduction.md](https://github.com/dotnet/docs/blob/main/docs/core/introduction.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **dotnetcontent**

**Comment by SeanKilleen** [comment]
I looked over the doc and it largely seems correct.

I found one instance of .NET 5 that should be `.NET 5+`, and did a PR to resolve that.

In most cases, the docs seem to (correctly IMO) show that .NET Core and .NET 5 are the same thing. In all the language I reviewed, that distinction was clear.

So I've noted that the PR for the one missing `+` sign should be considered to resolve this issue.

**Comment by gewarren** [comment]
@SeanKilleen I think this issue is more about changing .NET 5 wording to .NET 6, similar to this comment: https://github.com/dotnet/docs/pull/29671#issuecomment-1141266458. So I'm going to remove the "fixes" verbiage from your linked PR so this doesn't get closed.

**Comment by SeanKilleen** [comment]
@gewarren I would agree, except that in this case the references to `.NET 5` and `.NET 5+` seem to explicitly be in the context of helping people understand that .NET Core & .NET 5+ are the same thing, because .NET 5 is when the .NET core branding was retired and .NET took the path of .NET Core.

**Comment by tdykstra** [comment]
Closing, as the article was rewritten after this issue and its comments were written, and it only has one remaining reference to .NET 5, in the history section.
